examples/nixnet/ethernet-frame-stream-input-reader.py

```diff
@@ -0,0 +1,183 @@
+r""" Reads all the frame on network.
+
+  This example reads all the frames on the network and displays them in table format. 
```

Comment:
nit: Can be changed to "displays them in tabular format"

Reply:
Its not actually printing them in tabular format though... :)

Reply:
Okay, @kothariaman probably just check if it can be displayed in tabular format or else we can replace this statement from the comment.

Reply:
Replaced the comment . We are not printing it in tabular format.